policyengine-client/src/countries/uk/data/policy.jsx

```diff
@@ -123,7 +131,8 @@ export const PARAMETER_HIERARCHY = {
 			"autoUBI",
 		],
 		"Land Value Tax": [
-			"LVT",
+			"household_lvt",
```

Comment:
These are separate parameters? I'd prefer to expose one unless people request otherwise - base consistency is a LVT feature.

Reply:
Re-added the plain LVT here